templates/CRM/Contact/Page/DedupeFind.tpl

```diff
@@ -84,6 +190,206 @@
 {include file='CRM/common/dedupe.tpl'}
 {literal}
 <script type="text/javascript">
-var oTable = null;
+CRM.$(function($) {
+  var sourceUrl = {/literal}'{$sourceUrl}'{literal};
+  var context   = {/literal}'{$context}'{literal};
+  $('#dupePairs').DataTable({
+    //"scrollX": true, // doesn't work with hover popup for for icons
+    "lengthMenu": [[10, 25, 50, 100, 1000, 2000, -1], [10, 25, 50, 100, 1000, 2000, "All"]],
+    "searching" : true,
+    "dom": 'flrtip',
+    "processing": true,
+    "serverSide": true,
+    "ajax": sourceUrl,
+    "columns"  : [
+      {data: "is_selected_input"},
+      {data: "src_image"},
+      {data: "src"},
+      {data: "dst_image"},
+      {data: "dst"},
+      {data: "src_email"},
+      {data: "dst_email"},
+      {data: "src_street"},
+      {data: "dst_street"},
+      {data: "src_postcode"},
+      {data: "dst_postcode"},
+      {
+        data: "conflicts",
+        className: "crm-pair-conflict"
```

Comment:
You can do this simply by adding this class to the header e.g. `<td class="crm-pair-conflicts">` and it will be copied to each cell.